deferred-leveldown.js

```diff
@@ -1,13 +1,14 @@
 var util = require('util')
 var AbstractLevelDOWN = require('abstract-leveldown').AbstractLevelDOWN
 var DeferredIterator = require('./deferred-iterator')
-var deferrables = 'put get del batch approximateSize'.split(' ')
+var deferrables = 'put get del batch'.split(' ')
 
 function DeferredLevelDOWN (db) {
   AbstractLevelDOWN.call(this, '')
   this._db = db
   this._operations = []
   this._iterators = []
+  closed(this)
```

Comment:
Note that I moved this from the prototype to `this` since we need to know the properties of `db` to determine if `approximateSize` is present or not.